clang/include/clang/StaticAnalyzer/Core/PathSensitive/SMTConv.h

```diff
@@ -570,23 +572,37 @@ class SMTConv {
   // TODO: Refactor to put elsewhere
   static inline QualType getAPSIntType(ASTContext &Ctx,
                                        const llvm::APSInt &Int) {
-    return Ctx.getIntTypeForBitwidth(Int.getBitWidth(), Int.isSigned());
+    QualType Ty = Ctx.getIntTypeForBitwidth(Int.getBitWidth(), Int.isSigned());
```

Comment:
```suggestion
    if (QualType Ty = Ctx.getIntTypeForBitwidth(Int.getBitWidth(), Int.isSigned()); !Ty.isNull())
      return Ty;
```

Then deindent the rest of the code and drop the isNull check.

Reply:
I think this has been addressed, resolving.